src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -397,4 +411,19 @@ private long getLong(String name) {
     private String[] getStringArray(String name) {
         return mProperties.getStringArray(name);
     }
+
+    // This method can be useful if you use include and have several times the same key
+    // Retrieve the last children file value
+    @VisibleForTesting
+    String getStringLastValue(String name, String defaultValue) {
```

Comment:
I don't find any other method to override properties.
When we include a file into another, i want only the child properties but the default behaviour is to return the first value (the parent, included file).